docs/core/testing/unit-testing-platform-intro.md

```diff
@@ -183,56 +183,72 @@ To run a test project in CI add one step for each test executable that you wish
 
 The list below described only the platform options. To see the specific options brought by each extension, either refer to the extension documentation page or use the `--help` option.
 
+- **`@`**
+
+  Specifies a response file (rsp).
+
+- **`--config-file`**
+
+  Specifies a [*testconfig.json*](unit-testing-platform-config.md) file.
+
 - **`--diagnostic`**
 
-Enables the diagnostic logging. The default log level is `Trace`. The file is written in the output directory with the following name format, `log_[MMddHHssfff].diag`.
+  Enables the diagnostic logging. The default log level is `Trace`. The file is written in the output directory with the following name format, `log_[MMddHHssfff].diag`.
 
 - **`--diagnostic-filelogger-synchronouswrite`**
 
-Forces the built-in file logger to synchronously write logs. Useful for scenarios where you don't want to lose any log entries (if the process crashes). This does slow down the test execution.
+  Forces the built-in file logger to synchronously write logs. Useful for scenarios where you don't want to lose any log entries (if the process crashes). This does slow down the test execution.
 
 - **`--diagnostic-output-directory`**
 
-The output directory of the diagnostic logging, if not specified the file is generated in the default _TestResults_ directory.
+  The output directory of the diagnostic logging, if not specified the file is generated in the default _TestResults_ directory.
 
 - **`--diagnostic-output-fileprefix`**
 
-The prefix for the log file name. Defaults to `"log_"`.
+  The prefix for the log file name. Defaults to `"log_"`.
 
 - **`--diagnostic-verbosity`**
 
-Defines the verbosity level when the `--diagnostic` switch is used. The available values are `Trace`, `Debug`, `Information`, `Warning`, `Error`, or `Critical`.
+  Defines the verbosity level when the `--diagnostic` switch is used. The available values are `Trace`, `Debug`, `Information`, `Warning`, `Error`, or `Critical`.
+
+- **`--exit-on-process-exit`**
```

Comment:
@Evangelink I wanted to make sure if we want to document this one, or if it was intentionally missing.

Reply:
Good question, at the moment we mainly use it to ensure that if some other launching process terminates we also terminates. It feels more technical but maybe it's worth mentioning for users.

WDYT @MarcoRossignoli ?

Reply:
I would document it, it's public right? And could be useful if the test is started from some tool.